announcements/_posts/2019-11-15-w12-kernel-tracing.md

```diff
@@ -2,8 +2,8 @@
 author: Jeffery Russell (@jrtechs)
 title: "Kernel Tracing"
 layout: post-event
-date-start: "2019-10-18 16:30"
-date-end: "2019-10-18 18:00"
+date-start: "2019-11-18 16:30"
```

Comment:
Supposed to be November 15th
![kernel](https://user-images.githubusercontent.com/13894625/72026051-2a7b9e00-3248-11ea-8e79-0ca840c60af0.png)

Reply:
Ah, I just looked at the month. Give me a sec and I'll fix